addons/breathing/XEH_preInit.sqf

```diff
@@ -38,6 +38,26 @@ PREP_RECOMPILE_END;
     true
 ] call CBA_Settings_fnc_init;
 
+// Cardiac Arrest SpO2 value
+[
+    QGVAR(SpO2_cardiacValue),
+    "SLIDER",
+    LLSTRING(SETTING_SpO2_cardiacValue),
+    [CBA_SETTINGS_CAT, ELSTRING(GUI,SubCategory_Basic)],
+    [5, 95, 75, 0],
```

Comment:
minimal value as 0 is preferable as some people have strange settings
same with max as 100